File: westcoast/src/main/java/frc/commands/ballhandling/ShooterAccelerator.java
package frc.commands.ballhandling;

import edu.wpi.first.wpilibj2.command.CommandBase;
import frc.subsystems.BallHandlingSubsystem;

public class ShooterAccelerator extends CommandBase {
  private BallHandlingSubsystem ball;

  public ShooterAccelerator(
    BallHandlingSubsystem ballHandler
  ) {
    this.ball = ballHandler;
    addRequirements(ball);
  }

  @Override
  public void initialize() {
    ball.shooterTune();
  }

  @Override
  public void execute() {
    ball.runShooter(0.78);
    ball.getCurrentLevels();
  }

  @Override
  public void end(boolean interrupted) {
    ball.stopMotors();
    ball.setShooterPercent();
  }

  @Override
  public boolean isFinished() {
    return false;
  }
}

File: westcoast/src/main/java/frc/commands/ballhandling/ShooterAccelerator.kt
package frc.commands.ballhandling

import edu.wpi.first.wpilibj2.command.CommandBase
import frc.subsystems.BallHandlingSubsystem

class ShooterAccelerator(
    private val ball: BallHandlingSubsystem
) : CommandBase() {
  override fun initialize() {
    ball.shooterTune()
  }

  override fun execute() {
    ball.runShooter(0.78)
    ball.currentLevels
  }

  override fun end(interrupted: Boolean) {
    ball.stopMotors()
    ball.setShooterPercent()
  }

  override fun isFinished(): Boolean {
    return false
  }

  init {
    addRequirements(ball)
  }
}